kos/models.py

```diff
@@ -72,6 +72,18 @@ def has_team_passed(self, team):
         """Vráti bool či tím už šifru vyriešil"""
         return self.submissions.filter(team=team).aggregate(Max('correct'))['correct__max']
 
+    def team_timeout(self, team):
+        """Vráti čas, o ktorý bude daný tím môcť znova odovzdať túto úlohu"""
+        submission = self.team_submissions(team)
+        if submission.count() < 3:
+            return timedelta()
```

Comment:
Ja by som asi pre lepsiu citatelnost vlozil do tych timedelta explictne 0 .. bo mna to zmiatlo kusok